modules/core/src/lighting/lighting-effect.js

```diff
@@ -0,0 +1,26 @@
+export default class LightingEffect {
```

Comment:
API Audit: I think the idea is to have a `LightingEffect extends Effect` in deck.gl.  Also having a `LightingEffect` in luma.gl could be confusing?

As an alternative, perhaps we could just pass array(s) of the various light sources as parameter(s) to the lighting shader module?

Reply:
The Idea is to move Effect and LightingEffect classes to luma.gl, because eventually all the rendering effects will be in luma.gl as fundamental modules as we planed long time ago, is that right?

Reply:
Done, moved LightEffect to deck.gl as our discussion